.clang-format

```diff
@@ -61,44 +78,71 @@ ForEachMacros:
   - WT_MODIFY_FOREACH_REVERSE
   - WT_ROW_FOREACH
   - WT_SKIP_FOREACH
+IncludeBlocks:   Preserve
 IncludeCategories:
   - Regex:           '^"(llvm|llvm-c|clang|clang-c)/'
     Priority:        2
-  - Regex:           '^(<|"(gtest|isl|json)/)'
+    SortPriority:    0
+  - Regex:           '^(<|"(gtest|gmock|isl|json)/)'
     Priority:        3
+    SortPriority:    0
   - Regex:           '.*'
     Priority:        1
+    SortPriority:    0
+IncludeIsMainRegex: '(Test)?$'
+IncludeIsMainSourceRegex: ''
 IndentCaseLabels: false
+IndentGotoLabels: false
```

Comment:
This is the one we care about. Woo!
I was able to cleanly remove `s_goto.py` with no problems because of this.